eppo/build.gradle

```diff
@@ -189,11 +164,4 @@ tasks.withType(PublishToMavenRepository) {
     onlyIf {
         project.ext.has('shouldPublish') && project.ext.shouldPublish
     }
-}
-
-signing {
-    sign publishing.publications.release
-    if (System.env['CI']) {
-        useInMemoryPgpKeys(System.env.GPG_PRIVATE_KEY, System.env.GPG_PASSPHRASE)
-    }
-}
```

Comment:
According to the docs: "Signing is not required for snapshot builds, but if the configuration is present the build will still be signed."
https://vanniktech.github.io/gradle-maven-publish-plugin/central/#in-memory-gpg-key

So this should still behave the same as before